Content.Server/Chemistry/EntitySystems/ChemMasterSystem.cs

```diff
@@ -261,17 +278,55 @@ private void OnOutputToBottleMessage(Entity<ChemMasterComponent> chemMaster, ref
             ClickSound(chemMaster);
         }
 
+        // Begin DeltaV - chemmaster sources
+        private bool GetSourceSolution(
```

Comment:
make this return `Entity<SolutionComponent>?` as with the out ... you have to use .Value instead of just `if (GetSourceSolution(ent) is not {} source) return;` (also picking component instead of source is likely to conflict in the future)